jsapp/js/account/organization/MemberRoleSelector.tsx

```diff
@@ -25,6 +25,10 @@ export default function MemberRoleSelector(
       name={`member-role-selector-${username}`}
       type='outline'
       size='m'
+      // We assume that the row with the owner (who doesn't have
+      // `MemberRoleSelector`) will be first in the table. This means that all
+      // the other rows can open the selector up.
+      placement='up-center'
```

Comment:
I'm not sure we can assume this, unfortunately. It doesn't appear to be true on my local organization, where I have a member appearing above the owner. Additionally, we need to account for multiple pages. Even if the owner was always the first in the list of members, if an org 11 members, for example, the second page of results will only have one member and they will not be the owner.

Reply:
I think this is OK for now. There is still one annoying case where we run into trouble: An org of X + 1 members, where the table page size is set at X. Neither up nor down is going to work there. I am OK with this, however. We're really just doing a bandaid here pending implementation of a component library.